### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Bibin-Antony**
minor Bug fix

### CODE REVIEW

**Comment by Bibin-Antony** [comment]
https://stoic-banach-10962c.netlify.app/
Preview the website

File: styles/style.css [hunk 1]
```diff
@@ -64,23 +77,271 @@
...
+    color: var(--light-black);
+}
+.main-hero img{
+    width: 100%;
+    height: auto;
+}
+.hero--collection-text{
+    position: absolute;
+    display: flex;
+    flex-direction: column;
```
Comment (rohanmathur91) [inline styles/style.css:145]: have a utility class for this

**Review by biswanath-tewari — APPROVED** [review]

File: styles/style.css [hunk 2]
```diff
@@ -64,23 +77,271 @@
...
+    padding-left: 20%;
+}
+.hero-holder{
+    width: fit-content;
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+    padding: 4rem 2rem;
+    border-radius: 2rem;
+    background-color: rgba(200, 200, 200, 0.4);
```
Comment (rohanmathur91) [inline styles/style.css]: define a variable for this color and use that

**Review by rohanmathur91 — APPROVED** [review]
Added few comments, rest looks good

File: styles/style.css (hunk 1 above)
Comment (Bibin-Antony) [reply styles/style.css:145]: yes will do.

File: styles/style.css (hunk 2 above)
Comment (Bibin-Antony) [reply styles/style.css]: ok will do

**Review by hemanshthakur — APPROVED** [review]

**Pull request merged**